README.hi.md

```
@@ -0,0 +1,441 @@
+
+1. **आपका data आपके control में है।** आपका CV, contact info, और personal data आपकी machine पर रहता है और directly उस AI provider को भेजा जाता है जो आप choose करते हैं (Anthropic, OpenAI, आदि)। हम आपका कोई भी data collect, store, या access नहीं करते।
+2. **AI आपके control में है।** Default prompts AI को auto-submit applications के लिए instruct नहीं करते, लेकिन AI models अप्रत्याशित रूप से behave कर सकते हैं। यदि आप prompts modify करते हैं या अलग models use करते हैं, तो आप अपने risk पर करते हैं। **Submit करने से पहले accuracy के लिए AI-generated content हमेशा review करें।**
+3. **आप third-party ToS का पालन करते हैं।** आपको इस tool को उन career portals के Terms of Service के अनुसार use करना है जिनसे आप interact करते हैं (Greenhouse, Lever, Workday, LinkedIn, आदि)। Employers को spam करने या ATS systems को overwhelm करने के लिए इस tool का use न करें।
+4. **कोई guarantee नहीं।** Evaluations recommendations हैं, truth नहीं। AI models skills या experience hallucinate कर सकते हैं। Authors employment outcomes, rejected applications, account restrictions, या किसी अन्य consequences के लिए liable नहीं हैं।
+
+Full details के लिए [LEGAL_DISCLAIMER.md](LEGAL_DISCLAIMER.md) देखें। यह software [MIT License](LICENSE) के under "as is" provide किया जाता है, बिना किसी warranty के।
+
+## Contributors
+
+<a href="https://github.com/santifer/career-ops/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=santifer/career-ops" />
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)